atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -97,21 +102,40 @@ def generate_auto_labels(self, auto_labels):
 
         for lbl in auto_labels:
             if lbl in self.labels:
-                self.log.info("label %s is set explicitly, not using generated value", lbl)
+                self.log.info("label %r is set explicitly, not using generated value", lbl)
                 continue
 
             if lbl in generated:
                 self.labels[lbl] = generated[lbl]
             else:
-                self.log.warning("requested automatic label %s is not available", lbl)
+                self.log.warning("requested automatic label %r is not available", lbl)
+
+    def add_renamed_labels(self, base_labels, df_labels, new_labels):
```

Comment:
Worth pointing out in the docstring that `df_labels` is a mutable view of the labels in the Dockerfile, and that updating it writes out new Dockerfile content?

Also, as the original labels are left in place I does 'rename' really describe it well? They're aliases or copies, really.